public/static-checkout/css/style.css

```diff
@@ -0,0 +1,256 @@
+body {
```

Comment:
perhaps when we're done we could run a css linter that could help find the unused rules